README.md

```diff
@@ -226,22 +83,22 @@ Example:
 
 > Describe 1 - 3 features you will add to the project.
 
-- [ ] **[new_feature_1]**
-- [ ] **[new_feature_2]**
-- [ ] **[new_feature_3]**
+- [ ] **[Navigation Bar]**
+- [ ] **[Images]**
+- [ ] **[Responsiveness]**
 
```

Comment:
- [ ] Again, as my colleague opined, kindly consider removing the braces around your texts except it is a link (which will be immediately followed by a bracket) unlike this case. This allows readers to understand you and read your documentation with ease.